enhancements/api-review/add-ClusterImagePolicy-and-ImagePolicy-for-Signature-Verification.md

```diff
@@ -192,6 +197,38 @@ type FulcioCAWithRekor struct {
 	FulcioSubject PolicyFulcioSubject `json:"fulcioSubject,omitempty"`
 }
 
+// PKI defines the root of trust based on Root CA(s) and corresponding intermediate certificates.
+type PKI struct {
+	// CertificateAuthorityRootsData contains a certificate bundle PEM file containing one or more CA roots
+	// in the PEM format.
```

Comment:
I think `caRootsData` and  `caIntermediatesData`  are base64 encoded of the PEM format data? I will add a commit to address this.

Reply:
yes, thanks.

Reply:
(Documenting `[]byte` fields is a bit awkward — it’s base64-encoded _in the JSON form_ but not in the Go values.

When other fields document the base64 aspect, doing that here makes sense to me.)

Reply:
Thanks.